docs/going-into-production.rst

```diff
@@ -0,0 +1,353 @@
+overview of the two different ways to bootstrap a cluster.
+
+If you have been using CrateDB for development on your local machine, there is
+a good chance you have been using :ref:`single host auto-bootstrapping
+<auto-bootstrapping>`.
+
+For improved performance and resiliency, production CrateDB clusters should be
+run with one node per host machine. To do this, you must manually configure the
+bootstrapping process by telling nodes how to:
+
+  a. :ref:`Discover other nodes <discovery>`, and
+  b. :ref:`Elect a master node <master-node-election>`
+
+This is known as *manual bootstrapping*. See the :ref:`how-to guide
+<manual-bootstrapping>` for more information about how bootstrap a cluster
+manually.
+
+Switching to a manual bootstrapping configuration is the first step towards
+going into production.
+
+
+.. _prod-naming:
+
+Naming
+======
+
+
+.. _prod-cluster-name:
+
+Configure a logical cluster name
+--------------------------------
+
+The `cluster.name`_ setting allows you to override the default cluster name of
+``crate``. You should use this setting to give a logical name to your cluster.
+
+For example, add this to your `configuration`_ file:
+
+.. code-block:: yaml
+
+    cluster.name: acme-prod
+
+The ``acme-prod`` name suggests that this cluster is the production cluster for
+the *Acme* organization. If *Acme* has a cluster running in a staging
+environment, you might want to name it ``acme-staging``. This way, you can
+differentiate your clusters by name (visible in the `Admin UI`_).
+
+.. TIP::
+
+    A node will refuse to join a cluster if the respective cluster names
+    do not match
+
+.. SEEALSO::
+
+    :ref:`Cluster names for multi-node setups <multi-node-cluster-name>`
+
+
+.. _prod-node-names:
+
+Configure logical node names
+----------------------------
+
+By default, CrateDB sets the node name for you. You can override this by
+configuring a explicit node name using the `node.name`_ setting in your
+`configuration`_ file.
+
+Automatic nodes names are chosen the `sys.summits`_ table. This is useful in
+development when you don't care too much about cluster architecture. However,
+when going into production, you should give a logical name to each one of your
+nodes.
+
+CrateDB supports `multiple types of node`_, determined by the ``node.master``
+and ``node.data`` settings. You can use this to name your nodes according to
+their type. For instance:
+
+.. SEEALSO::
+
+    :ref:`Node names for multi-node setups <multi-node-node-name>`
+
+
+.. _prod-node-md:
+
+Multi-purpose nodes
+~~~~~~~~~~~~~~~~~~~
+
+A master-eligible node that also handles query execution loads can be
+configured like this:
+
+.. code-block:: yaml
+
+    node.master: true
+    node.data: true
+
+A good name for this node might be:
+
+.. code-block:: yaml
+
+    node.name: node-01-md
+
+Here, ``node`` is used as a base name with a sequence number of ``01``. Every
+node in the cluster should have a unique sequence number, irrespective of node
```

Comment:
I feel like "irrespective" might be a weasel word but am not sure.

Reply:
switched to "independent"